cmd/daprd/options/options.go

```diff
@@ -117,8 +119,10 @@ func New(origArgs []string) (*Options, error) {
 	fs.StringVar(&opts.DaprHTTPPort, "dapr-http-port", strconv.Itoa(runtime.DefaultDaprHTTPPort), "HTTP port for Dapr API to listen on")
 	fs.StringVar(&opts.DaprAPIListenAddresses, "dapr-listen-addresses", runtime.DefaultAPIListenAddress, "One or more addresses for the Dapr API to listen on, CSV limited")
 	fs.StringVar(&opts.DaprPublicPort, "dapr-public-port", "", "Public port for Dapr Health and Metadata to listen on")
+	fs.StringVar(&opts.DaprPublicListenAddress, "dapr-public-listen-address", "0.0.0.0", "Public listen address for Dapr Health and Metadata")
```

Comment:
```suggestion
	fs.StringVar(&opts.DaprPublicAddress, "dapr-public-address", "0.0.0.0", "Public listen address for Dapr Health and Metadata")
```

Reply:
I personally like having `Listen` in the name otherwise it's not explicit to developers setting it without looking at the description in this one file or via the cli description.

Reply:
How about the `DaprPublicPort`, I prefer to keep same.

Reply:
I agree- `address` without `listen` gives the impression of a target host. We are already using the `listen` key word in an existing flag so this also keeping consistent. https://github.com/dapr/dapr/blob/8bd7e07a852a4d0727e518563d300bdb5d92a34f/cmd/daprd/options/options.go#L118